dashboardsly/static/css/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Ignore everything except this gitignore
+*
+!.gitignore
```

Comment:
Required so that the directories are checked into git and the build succeeds on Heroku - otherwise `browserify` complains about a non-existent directory.